### PR TITLE
Update to itwinui-css 0.24.1

### PR DESCRIPTION
**Pull request by mayank99**
- Bumped itwinui-css from 0.23.0 to 0.24.1
- Added class to menu item
- Updated unit tests
- Updated images

### CODE REVIEW

File: src/core/Select/Select.tsx [hunk 1]
```diff
@@ -270,11 +270,10 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
       <DropdownMenu
         menuItems={menuItems}
         placement='bottom-start'
-        className={menuClassName}
+        className={cx('iui-scroll', menuClassName)}
```
Comment (veekeys) [inline src/core/Select/Select.tsx:273]: Why we needed separate class for this? Wouldnt it make sense that its always scrollable?

Comment (mayank99) [reply src/core/Select/Select.tsx:273]: There must be some reason that previously we didn't even have it in the css.

Maybe Select makes sense to always be scrollable, but not other Menus.

Comment (veekeys) [reply src/core/Select/Select.tsx:273]: which ones not?

Comment (mayank99) [reply src/core/Select/Select.tsx:273]: idk, just a normal menu.

I'm not seeing the problem here. All I did was move `overflow` from inline style into a css class.
image

We could make it default but that will be a separate css release.

**Review by veekeys — APPROVED** [review]

**Pull request merged**